src/components/common/Actions.vue

```diff
@@ -206,6 +208,9 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
    * @returns a promise (ie, this is an async method)
    */
   private async onClickFilePay (): Promise<void> {
+    // Set the status of clickFileAndPay for registration
+    if (this.isTypeFirm) this.setClickFileAndPay(true)
```

Comment:
I'm surprised this ticket/fix applies to firms only. Please confirm with UI/UX.

On the other hand, if this bug is only seen with firms then there's something wrong with the firms code that should be fixed directly instead of changing the app's architecture like this.